COLLADAMaya/include/COLLADAMayaSyntax.h

```diff
@@ -38,10 +38,13 @@ namespace COLLADAMaya
     static const char* PROFILE_COLLADA                              = "COLLADA";
     static const char* PROFILE_MAYA                                 = "OpenCOLLADAMaya";
     static const char* PROFILE_MAX                                  = "OpenCOLLADA3dsMax";
+	static const char* PROFILE_FL4RE								= "fl4re";
```

Comment:
We talked about this, but I don't see a change. 
can you send me an email to discuss what you want to happen on this PR please?
Or just comment here.